### ISSUE TITLE
log file is not saved after import

### ISSUE THREAD

**Issue by setphen** [joshavanier/log#46]
When I import a log, and then quit the app, it is not saved. After import, it should write to the app's log storage 😸

**Comment by Godje** [comment]
Which OS are you on?
Are you opening the app through Itch.io, manual build or download?

**Comment by setphen** [comment]
macOS build of 1.0.4 from itch.io. It has no problem writing the file when I create new entries.

**Comment by joshavanier** [comment]
When you import a file, the app starts reading from and writing to that file. It used to store everything in localStorage but apparently that gets overwritten if you update the app. By default, if you don't import anything and just start using the app, your data is stored in a file in `appData` which on Mac should be `~/Library/Application Support`

**Comment by joshavanier** [comment]
Wait, did you import by dragging and dropping? I just noticed that it doesn't store the location of the dragged-and-dropped file and so the app won't write anything to it.

**Comment by setphen** [comment]
Yeah, it was a drag and drop. I see what you mean about recording data to the new location when you do an import. Thanks!

**Comment by joshavanier** [comment]
Oh cool. Alright I'll see if I have time to fix that today :)
